src/script/bitcoinconsensus.h

```diff
@@ -70,6 +80,11 @@ EXPORT_SYMBOL int bitcoinconsensus_verify_script_with_amount(const unsigned char
                                     const unsigned char *txTo        , unsigned int txToLen,
                                     unsigned int nIn, unsigned int flags, bitcoinconsensus_error* err);
 
+EXPORT_SYMBOL int bitcoinconsensus_verify_script_with_spent_outputs(const unsigned char *scriptPubKey, unsigned int scriptPubKeyLen, int64_t amount,
+                                    const unsigned char *txTo        , unsigned int txToLen,
+                                    UTXO *spentOutputs, unsigned int spentOutputsLen,
```

Comment:
Same as for the other two pointer parameters (`scriptPubKey` and `txTo`), I think `spentOutputs` should also follow const-correctness for consistency and type-safety reasons (otherwise this suggests to the caller that the passed `UTXO` instance could be modified inside):
```suggestion
                                    const UTXO *spentOutputs, unsigned int spentOutputsLen,
```

Reply:
done